CONTRIBUTING.md

```diff
@@ -68,3 +68,30 @@ An existing Android emulator is required to match the name defined in `detox.con
   yarn detox:android:build:release
   yarn detox:android:test:release
   ```
+
+### Fabric
+
+Fabric is the new React Native rendering system ([read more about it here](https://reactnative.dev/architecture/fabric-renderer)).
```

Comment:
I'd love to see fabric being used in CI (e2e tests) - have you tried it? Thanks

Reply:
It seems Detox does not support new arch yet